raytracing/imagingpath.py

```diff
@@ -310,6 +310,13 @@ def displayRange(self):
             for (planePosition, magnification) in conjugates:
                 if displayRange < self.objectHeight * magnification:
                     displayRange = self.objectHeight * magnification
+
+        if axes is not None:
+            if len(axes.lines) != 0:
+                for rayTrace in axes.lines:
+                    if max(abs(rayTrace._y)) * 2 > displayRange:
+                        displayRange = max(abs(rayTrace._y)) * 2
+
         return displayRange
 
     def createFigure(self, limitObjectToFieldOfView=False,
```

Comment:
Change of API. Tut, tut, tut... Here is a video of the last person who did that: https://youtu.be/GTQAXX08A-s?list=WL&t=21
We should discuss any change of API.

Reply:
it's going down